cmd/cagent/toast_fallback.go

```diff
@@ -0,0 +1,21 @@
+// +build !windows
+
+package main
+
+import (
+	"errors"
+
+	"github.com/cloudradar-monitoring/cagent"
+)
+
+func sendErrorNotification(_, _ string) error {
```

Comment:
Why do we need these functions?

Reply:
this is fallback for non-win, because this function should be called in the `main.go`

Reply:
Solved?